### DISCUSSION TITLE
【Bug】位图预合成混合模式后不显示，是否与图像计算时序相关

### DISCUSSION THREAD

**Discussion by HankeAvailable** [Tencent/libpag#1969] [General]
1. 两张位图需要导出单独bmp
2. bmp合成图层顺序在上，使用相加混合模式，被混合图层在下，结果：bmp图层正常导出
3. 普通图层在上，使用相加混合模式，bmp合成在下，结果：bmp图层未导出

![微信截图_20231207175000](https://github.com/Tencent/libpag/assets/93767920/6ec0a521-5c85-4928-b4a0-39c513d65c69)
![微信截图_20231207175147](https://github.com/Tencent/libpag/assets/93767920/5f765a4b-78d3-46f8-b990-ca1fc2460e94)

AE version:2023.23.2.1
Pag version:4.38 beta

**Comment by CodeJhF** [comment]
抱歉这条当时没有被跟进到，是我们漏了。

因长期无回复，且 libpag 4.3 及更早版本已停止维护（当前仅维护 4.4 / 4.5），先关闭本条。
上报版本：4.38。

如果问题在最新版本（4.4 / 4.5）仍能复现，欢迎在本帖回复或新开 issue，并附上 libpag 版本号、平台与系统版本、可复现的 PAG 文件或最小 demo，我们会优先跟进。（stale-close）
